.github/workflows/sv_pipeline_docker.yml

```diff
@@ -1,44 +1,71 @@
 name: Build Docker Images
 
-on: 
+on:
   push:
     branches:
       - master
     paths:
       - 'src/**'
       - 'dockerfiles/**'
+      - 'scripts/**'
+      - '.github/workflows/sv_pipeline_docker.yml'
```

Comment:
Why were these added? (same below?)

Reply:
A change on this file can impact the image build and push to GCR. For instance, if we would like to change how we tag the images, or a PR may change how the `build_docker` is called and break building images. Without adding this file to the `paths` list, any such PRs (or pushes) will not trigger building and pushing the images.

Reply:
I see - can you change `scripts/**` to `scripts/docker/build_docker.py` and `scripts/docker/resources.Dockerfile`? Those are the only files that should affect the build.

Reply:
Good point, thanks. Changed.